radioapp/ui/ui_backlight.py

```diff
@@ -0,0 +1,62 @@
+from tkinter import *
+
+
+class UIBackLight():
+	"""Creates UI Backlight Screen."""
+
+	def __init__(self):
+		"""Creates a window, and adds slider and label for UI Backlight."""
+
+		self.window = Tk()
+		self.window.title("Backlight Setting")
+		self.window.geometry("160x128")
+		self.window.configure(background = "#ff9966")

```

Comment:
Though you shouldn't need to set anything on the window (since UIManager already sets it up), this is a different color (cream orange) from the rest of the UI (blue).